examples/network/tcp_server_poll.py

```diff
@@ -0,0 +1,39 @@
+
+import select, socket
```

Comment:
There're no such modules in MicroPython, see the docs.

Reply:
well,`import select, socket` is the same as `import uselect, usocket  `, which is passed in the latest firmware.
Is there another reason to use uselect without select?